.github/workflows/run-tests.yml

```
@@ -0,0 +1,75 @@
+            -resultBundlePath ./ci-outputs/test-results/all-tests/TestResults.xcresult \
+            | xcbeautify --report junit --report-path ./ci-outputs/test-results/all-tests/junit.xml
+      - name: Upload Test Results
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: all-test-results-${{ github.run_id }}
+          path: |
+            ci-outputs/test-results/all-tests/TestResults.xcresult
+            ci-outputs/test-results/all-tests/junit.xml
+          retention-days: 7
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)